src/Model/Item.php

```diff
@@ -2259,7 +2259,7 @@ public static function expire(int $uid, int $days, string $network = "", bool $f
 		$condition[0] .= " AND `received` < UTC_TIMESTAMP() - INTERVAL ? DAY";
 		$condition[] = $days;
 
-		$items = Post::select(['resource-id', 'starred', 'id', 'post-type', 'uid', 'uri-id'], $condition);
+		$items = Post::select(['resource-id', 'starred', 'id', 'post-type', 'uid', 'uri-id', 'post-type'], $condition);
```

Comment:
`post-type` is already a part of the field list.

Reply:
aahh .. it was too late yesterday .. I totaly oversaw it ^^, thx

Reply:
I think I'm lost at this point ..

we select the `post-type`:
https://github.com/friendica/friendica/blob/2021.09-rc/src/Model/Item.php#L2262

We fetch one item
https://github.com/friendica/friendica/blob/2021.09-rc/src/Model/Item.php#L2283

Even in case it's not found, we would have failed here first (before we are trying to access the empty `$item['post-type']`):
https://github.com/friendica/friendica/blob/2021.09-rc/src/Model/Item.php#L2285

I think I do have to fake a call with missing post-type to verify what's happening ..
Meanwhile I revert this part to fix at least the other issues ..

Reply:
Yeah, I would vote for reverting only that part. To which issue does this refer to? There must be some underlying cause that should be fixed.

Reply:
https://github.com/friendica/friendica/issues/10474#issuecomment-893026960 <-- this is the issue

Reply:
Since the reported version is `2021.06-dev`, I guess that it had already been fixed in the current version, otherwise it would had been reported by others as well.